### ISSUE TITLE
Crash by clicking on one specific entry

### ISSUE THREAD

**Issue by MartinX3** [moneymanagerex/moneymanagerex#4042]
MMEX version:
 - [x] 1.5.10
 - [ ] 1.6.x

Operating System:
 - [] Windows
 - [ ] Mac OSX
 - [x] Linux 

# Description of the bug

It crashes if I filter and click on a specific entry.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

1. Filtering with *value*
2. Clicking on a specific entry
3. Crash

Expected result:
To view my specific entry.

Actual result:
```
Pango:ERROR:../pango/pango/pango-layout.c:4507:process_line: assertion failed: (result == BREAK_SOME_FIT || result == BREAK_EMPTY_FIT)
Bail out! Pango:ERROR:../pango/pango/pango-layout.c:4507:process_line: assertion failed: (result == BREAK_SOME_FIT || result == BREAK_EMPTY_FIT)
Abort (Segmentation fault)
```

# Additional information

```
Money Manager Ex
Version: 1.5.10 64-bit
• Aufbauend auf Nov 27 2021 18:06:08
• Datenbankversion: 7 • (aes128cbc)
• Git commit: 4b8bb1745 (2021-11-23)
MMEX verwendet folgene Unterstützungsprodukte:
• wxWidgets 3.1.5 (wxGTK 3.24)
• wxSQLite3 4.6.0 (SQLite 3.31.1)
• RapidJSON 1.1.0
• Lua 5.4.3
• lunasvg v2.0.1
• libcurl/7.80.0 OpenSSL/1.1.1l zlib/1.2.11 brotli/1.0.9 zstd/1.5.0 libidn2/2.3.2 libpsl/0.21.1 (+libidn2/2.3.0) libssh2/1.10.0 nghttp2/1.46.0
• gettext 0.21
• apexcharts.js
Genutztes Build:
• CMake 3.22.0
• GNU Make 4.3
• GCC 11.1.0 • EndeavourOS rolling 'rolling'
Läuft auf:
• EndeavourOS Linux "rolling"
• Linux 5.15.5-zen1-1-zen x86_64
• German (UTF-8)
• 1600x900 24bit 96x96ppi
```

**Comment by vomikan** [comment]
Related issues #3926  #3212  #2483 #2470 #1420

**Comment by XidRanReb** [comment]
> I was looking for the possible problematic commit.
> 
> If it can help, [cf7f479](https://github.com/moneymanagerex/moneymanagerex/commit/cf7f47975eb9b3e1671e5964a6c6354e7963be99) seems to introduce the bug.

I just notice that the same commit as #3926 introduce a bug on the Report > Transaction Report.
MMEX crash after the validation of the filter.

adding @whalley in the loop

**Comment by XidRanReb** [comment]
I just compiled a fresh 1.5.10 and can't reproduce the bug :thinking: 

Don't know how I did to reproduce yesterday :man_shrugging:

**Comment by MartinX3** [comment]
It seems I can only trigger the crash if I filter the view before.

**Comment by MartinX3** [comment]
Fixed by pango 1.50.1.
1.50.0 had a crash bug with specific fonts.
